Assets/MixedRealityToolkit/_Core/Managers/MixedRealityManager.cs

```diff
@@ -114,6 +114,10 @@ public void ResetConfiguration(MixedRealityConfigurationProfile profile)
 
         #endregion Mixed Reality runtime component registry
 
+        #region Mixed Reality scene object registry
+        public List<IMixedRealitySceneObject> MixedRealitySceneObjects { get; set; } = new List<IMixedRealitySceneObject>();
```

Comment:
read only list?

Reply:
Err, no.  This is constantly updated as new Scene objects are added to / destroyed in the scene.
Unless I fundamentally misunderstand what a Readonly list does :D

Reply:
Just that there's only ever one list, created when the MRM is created.

Reply:
Not so keen on the name either. `MixedRealitySceneGameObjects`?

`object != Object != GameObject`